pallets/xcmp-queue/src/lib.rs

```diff
@@ -145,7 +153,7 @@ pub mod pallet {
 		/// Suspends all XCM executions for the XCMP queue, regardless of the sender's origin.
 		///
 		/// - `origin`: Must pass `ControllerOrigin`.
-		#[pallet::weight(T::DbWeight::get().writes(1))]
+		#[pallet::weight((T::DbWeight::get().writes(1), DispatchClass::Operational,))]
```

Comment:
How is that ensured that this is only one write? `ensure_origin` could do any kind of operation. Same applies for the others.

Reply:
The right answer is to benchmark these methods too? Ok I can do that - that will mean they're all bench-marked which is great for consistency.

Reply:
Yes this should also be benchmarked.